README.md

```diff
@@ -2,6 +2,12 @@
 
 The aim of this tool is to predict an optimal stopping point for CI tests based on their test duration (runtimes). We perform an initial data analysis as well as feature engineering on the CI test data. Furthermore, we also calculate the optimal stopping point by identifying the distribution of the test duration values for different CI tests and comparing the distributions between the passing and failing tests.
 
+## AI4CI: AI for Continuous Integration
+
+This tool is part of a wider project that we have been working on called [AI4CI](https://github.com/aicoe-aiops/ocp-ci-analysis). AIOps is a critical component of supporting any Open Hybrid Cloud infrastructure. As the systems we operate become larger and more complex, intelligent monitoring tools and response agents will become a necessity.
+
+The problem that we are trying to address is that there is a need for automated monitoring, altering, AI driven analysis for operations and testing systems. Open source testing data originating from real world production systems is a rarity for public datasets. Also, we noticed a lack of AI driven metrics for open source community health. This brings us to the opportunity of leveraging datasets that are made available by running open source software and applications in production like openshift-ci, kubernetes, fedora and develop a collection of open source data science tools and AI models to collect and analyze the CI/CD data. Thus, enabling AI for CI. 
+
 ## Data Collection
 
 We collect historical test data mainly the test duration values from running workflows on Github and store the data on s3 storage.
```

Comment:
The approach for calculating OSP has been revised, can we change the sections below to the reflect the updated approach? Approximating distributions and finding goodness of fit is no longer our primary approach so maybe we can add a section on previous experimental methods and link the notebook from there?

Reply:
Thanks for pointing that out. Update it 👍